docs/feature_flag_configuration.md

```diff
@@ -6,24 +6,43 @@ The `FeatureFlagConfiguration` version `v1alpha2` CRD defines the a CR with the
-          value: "8080"
-    featureFlagSpec: 
-        flags: 
-            foo: 
-                state: "ENABLED"
-                variants: 
-                    bar: "BAR"
-                    baz: "BAZ"
-                defaultVariant: "bar"
+  featureFlagSpec:
+    flags:
+      foo:
+        state: "ENABLED"
+        variants:
+          bar: "BAR"
+          baz: "BAZ"
+        defaultVariant: "bar"
 ```
 
-Within the CRD there are 2 main objects, namely the `flagDSpec` and the `featureFlagSpec`, both offering a different set of configuration for the injected `flagd` sidecars.
+Within the CRD there are 2 main objects, namely the `featureFlagSpec` and the `syncProvider` each offering a different set of configurations for the injected `flagd` sidecars.
+
+## featureFlagSpec
 
 The `featureFlagSpec` is an object representing the flag configurations themselves, the documentation for this object can be found [here](https://github.com/open-feature/flagd/blob/main/docs/configuration/flag_configuration.md).
 
-The `flagDSpec` has one property called `envs` which contains a list of environment variables used to override the default start command values in `flagd`, the documentation for this configuration can be found [here](https://github.com/open-feature/flagd/blob/main/docs/configuration/configuration.md). These environment variables are also made available in the workload `Pod` for simplified configuration.
\ No newline at end of file
+## syncProvider
+
+The `syncProvider` specifies how the flag configuration will be supplied to flagd. It contains 2 optional members, `name` and `httpSyncConfiguration`.
+
+The default `syncProvider` is `kubernetes`, but can be configured globally at installation time via by modifying the `defaultSyncProvider` parameter or by setting the `sidecarConfiguration.defaultSyncProvider` helm value (`helm upgrade -i ofo openfeature/open-feature-operator  --set  sidecarConfiguration.defaultSyncProvider=filepath`).
+
+### name
+
+| Value                  | Explanation                                                                                                                                                                                                                                                                                                |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| "kubernetes" (default) | Instruct flagd to query the Kubernetes API directly for the flag configuration custom resource(s) specified in the workload annotation (`openfeature.dev/featureflagconfiguration`). This configuration requires the flag sidecar (and therefor the workload pod) to be able to access the Kubernetes API. |
```

Comment:
```suggestion
| "kubernetes" (default) | Instruct flagd to query the Kubernetes API directly for the flag configuration custom resource(s) specified in the workload annotation (`openfeature.dev/featureflagconfiguration`). This configuration requires the flag sidecar (and therefore the workload pod) to be able to access the Kubernetes API. |
```

Reply:
https://www.grammarly.com/blog/therefore-vs-therefor/#:~:text=Therefore%20is%20an%20adverb%20that,%E2%80%9D%20or%20%E2%80%9Cfor%20it.%E2%80%9D 

Dammit, you're right.